SCClassLibrary/Common/Collections/String.sc

```diff
@@ -529,4 +529,7 @@ String[char] : RawArray {
 		^this.primitiveFailed
 	}
 
+	asRational {
+		^Rational(this.split[0].asInteger, this.split[1].asInteger)
```

Comment:
better make the delimiter explicit. The default `/` was used (I guess) because it was used for paths. Also it is inefficient to split twice.